eng/native/init-vs-env.cmd

```diff
@@ -50,14 +50,9 @@ if "%VisualStudioVersion%"=="16.0" (
     set __PlatformToolset=v142
     goto :SetVCEnvironment
 )
-if "%VisualStudioVersion%"=="17.0" (
```

Comment:
I think this should stay to enable VS2022 dogfooding via running from VS2022 command prompt.